aspnetcore/signalr/configuration.md

```diff
@@ -375,8 +377,10 @@ In the JavaScript Client, these options can be provided in a JavaScript object p
 ```javascript
 let connection = new signalR.HubConnectionBuilder()
     .withUrl("/chathub", {
+        // "Foo: Bar" will not be sent with WebSockets or Server-Sent Events requests
+        headers: { "Foo": "Bar" },
         skipNegotiation: true,
-        transport: signalR.HttpTransportType.WebSockets
+        transport: signalR.HttpTransportType.LongPolling 
```

Comment:
I switched this from WebSockets to LongPolling. I want the header configuration to have an impact, and since skipNegotiation is true, WebSockets and SSE are out.

Reply:
Well now the example will just throw, `skipNegotiation` only works with WebSockets. In fact it only works when you explicitly specify `transport: signalR.HttpTransportType.WebSockets`

Reply:
Good catch. So do we remove headers or skipNegotiation from this sample?

Reply:
I vote for skipNegotiation to be removed. Also, should be removed from the .NET one above too, unless you add WebSockets.